src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Slider, Rail, Handles, Tracks, Ticks } from 'react-compound-slider';
+import { SliderRail, Handle, Track, Tick } from './component';
+
+const sliderStyle = {
+    position: 'relative' as const,
```

Comment:
What was the error you were getting without this typecast?

Reply:
(JSX attribute) SliderProps.rootStyle?: React.CSSProperties | undefined
No overload matches this call.
  Overload 1 of 2, '(props: SliderProps, context?: any): ReactElement<any, any> | Component<SliderProps, any, any> | null', gave the following error.
    Type '{ position: string; width: string; touchAction: string; }' is not assignable to type 'CSSProperties'.
      Types of property 'position' are incompatible.
        Type 'string' is not assignable to type '"relative" | "-moz-initial" | "inherit" | "initial" | "revert" | "unset" | "-webkit-sticky" | "absolute" | "fixed" | "static" | "sticky" | undefined'.
  Overload 2 of 2, '(props: PropsWithChildren<SliderProps>, context?: any): ReactElement<any, any> | Component<SliderProps, any, any> | null', gave the following error.